### ISSUE TITLE
TypeError: this.jqEl is undefined

### ISSUE THREAD

**Issue by Treppenhouse** [primefaces/primefaces#3463]
## 1) Environment
- This error occurs on PrimeFaces version: 6.2
- This error did not occur on Primefaces version: 6.1
- Application server + version: Tomcat 7
- Affected browsers: Firefox, IE 11

## 2) Problem
It is hard for me to give a minimal example or describe steps to reproduce the problem. But I thought maybe you can already use the information given here and find the problem in the code. If not, I will try to investigate further.

This happened only after we upgraded to primefaces 6.2 (from 6.1). With 6.1 this error did not occur.

After clicking a button, the following error pops up in the firefox console:

```
TypeError: this.jqEl is undefined

components.js:13:4170
```

**Comment by tandraschko** [comment]
Impossible with this infos.

**Comment by melloware** [comment]
The firefox or Google Chrome console should have more information than that including a stack trace.

**Comment by tandraschko** [comment]
Will close it for now.

**Comment by Treppenhouse** [comment]
Some more context:

We have a `<p:commandButton>` in every line of a  `<p:dataTable>` to show a `<p:confirmDialog>` using `onclick="PF('abortRequestDialog').show();`. 

Inside that dialog there is a OK button that looks like this: 

    <p:commandButton value="OK" action="#{viewBean.abort()}" onstart="start()"
       oncomplete="stop();PF('abortRequestDialog').hide()" update="@form"
       process="abortPanel" ajax="false"/>`

The functions start() and stop() are showing some kind of progress widget and are defined like this:
```
<script type="text/javascript">
                function start() {
                    PF('statusDialog')
                            .show();
                }

                function stop() {
                    PF('statusDialog')
                            .hide();
                }
</script>
```

After clicking OK the abortRequestDialog closes. So far so good! However, when clicking on a button to open the dialog of this row or another row again, no dialog opens and the console logs this JS error:
```
TypeError: this.jqEl is undefined

components.js:13:4170
show
https://localhost:8443/MyApp/faces/javax.faces.resource/components.js:13:4170
onclick/<
https://localhost:8443/MyApp/faces/views/request_overview.xhtml:1:44
bcn
https://localhost:8443/MyApp/faces/javax.faces.resource/core.js:1:7837
onclick
https://localhost:8443/MyApp/faces/views/request_overview.xhtml:1:1
```



Anyway, after I set `ajax="true"` it now works with seemingly no problems.

**Comment by tandraschko** [comment]
I would really like to fix it - also as it doesn't occur in 6.1 - but i can't do it without a small replicator.

**Comment by melloware** [comment]
Fixed for 6.2.2 https://github.com/primefaces/primefaces/commit/9f86efba16ead70f9db1194744d291a7f64acefb
